### ISSUE TITLE
Safari - New Item dialog doesn't show up until you click on extension

### ISSUE THREAD

**Issue by ArchdukeTim** [bitwarden/clients#9950]
### Steps To Reproduce

1. Go to a fillable form
2. Click on autofill bit warden logo
3. Click ' + New Item'
4. Click on Bitwarden extension icon in menu bar

### Expected Result

After step 3, the new item dialog should appear

### Actual Result

The dialog only appears after step 4

### Screenshots or Videos

https://github.com/bitwarden/clients/assets/6132901/16076ef4-5488-443e-9b0b-c9b8a5e16690

### Operating System

macOS

### Web Browser

Safari

### Build Version

2024.3.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by ArchdukeTim** [comment]
Duplicate of #8958
